RecoTracker/LSTCore/src/alpaka/LSTEvent.dev.cc

```diff
@@ -371,12 +366,9 @@ void LSTEvent::createMiniDoublets() {
     alpaka::memset(queue_, totOccupancyMDs_view, 0u);
   }
 
-  Vec3D const threadsPerBlockCreateMD{1, 16, 32};
-  Vec3D const blocksPerGridCreateMD{1, nLowerModules_ / threadsPerBlockCreateMD[1], 1};
-  WorkDiv3D const createMiniDoublets_workDiv =
-      createWorkDiv(blocksPerGridCreateMD, threadsPerBlockCreateMD, elementsPerThread);
+  auto const createMiniDoublets_workDiv = cms::alpakatools::make_workdiv<Acc2D>({nLowerModules_ / 16, 1}, {16, 32});
```

Comment:
IIUC, the `16` in the grid and threads are (preferentially, I hope, rather than required) coupled. So, better add some constexpr value = 16

Reply:
That's true, I'll do that